### PR TITLE
lowercase the config

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
* [x] update the config to be lowercase
* [x] update the test config
* [x] update everywhere configs are mentioned

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,7 +55,6 @@ pylint:
 
 pytest:
 	@docker-compose up -d
-	@sleep 2
```
Comment (ThomasLaPiana) [inline Makefile:58]: this is not windows portable

**Comment by ThomasLaPiana** [comment]
a new release will need to get but after this PR to update fidesops and fidesctl

**Review by sanders41 — APPROVED** [review]
Thanks for taking this one @ThomasLaPiana!

We want to get #51 merged and into this release also. I'm going to review that one now and then we can do the release.

**Comment by ThomasLaPiana** [comment]
@sanders41 thanks!

**Pull request merged**